plugins/codesample.md

```diff
@@ -57,11 +53,11 @@ tinymce.init({
 });
 ```
 
-##### Using Prism.js on your web page
+### Using Prism.js on your web page
```

Comment:
Should this be a level 2 heading, or even be below the paragraph about Prism above?

Reply:
Fixed. Moved the section up, below the note about prism.